src/Symfony/Bridge/Twig/Mime/NotificationEmail.php

```diff
@@ -0,0 +1,220 @@
+        }
+
+        $this->context['markdown'] = true;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function content(string $content, bool $raw = false)
+    {
+        $this->context['content'] = $content;
+        $this->context['raw'] = $raw;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function action(string $text, string $url)
+    {
+        $this->context['action_text'] = $text;
+        $this->context['action_url'] = $url;
+
+        return $this;
+    }
+
+    /**
+     * @return self
+     */
+    public function importance(string $importance)
+    {
+        $this->context['importance'] = $importance;
+
+        return $this;
+    }
+
+    /**
+     * @param \Throwable|FlattenException
+     *
+     * @return $this
+     */
+    public function exception($exception)
+    {
+        $exceptionAsString = $this->getExceptionAsString($exception);
+
+        $this->context['exception'] = true;
+        $this->attach($exceptionAsString, 'exception.txt', 'text/plain');
+        $this->importance(self::URGENT);
+
+        if (!$this->getSubject()) {
+            $this->subject($exception->getMessage());
+        }
+
+        return $this;
+    }
+
+    public static function setDefaultTemplates(string $htmlTemplatePath = null, string $textTemplatePath = null)
+    {
+        if ($htmlTemplatePath) {
+            self::$htmlTemplatePath = $htmlTemplatePath;
+        }
+
+        if ($textTemplatePath) {
+            self::$textTemplatePath = $textTemplatePath;
+        }
+    }
+
+    public function getTextTemplate(): ?string
+    {
+        if ($template = parent::getTextTemplate()) {
+            return $template;
+        }
+
+        return self::$textTemplatePath;
+    }
+
+    public function getHtmlTemplate(): ?string
+    {
+        if ($template = parent::getHtmlTemplate()) {
+            return $template;
+        }
+
+        return self::$htmlTemplatePath;
+    }
+
+    public function getContext(): array
+    {
+        return array_merge($this->context, parent::getContext());
+    }
+
+    public function getPreparedHeaders(): Headers
+    {
+        $headers = parent::getPreparedHeaders();
+
+        $importance = $this->context['importance'] ?? self::LOW;
+        $this->priority($this->determinePriority($importance));
+        $headers->setHeaderBody('Text', 'Subject', sprintf('[%s] %s', strtoupper($importance), $this->getSubject()));
+
+        return $headers;
+    }
+
+    private function determinePriority(string $importance): int
+    {
+        switch ($importance) {
+            case self::URGENT:
+                return self::PRIORITY_HIGHEST;
+            case self::HIGH:
+                return self::PRIORITY_HIGH;
+            case self::MEDIUM:
+                return self::PRIORITY_NORMAL;
+            case self::LOW:
+                return self::PRIORITY_LOW;
+            default:
```

Comment:
Exactly, just defensive programming, but this should indeed never happen for now.

Reply:
then, I suggest using `self::PRIORITY_LOW` for this defensive path, so that this path does not unlock using a lowest priority than support code paths (and then having people wondering how they can trigger that code path without bugs elsewhere)